docs/content/api-overview/resources/aks-cluster.md

```diff
@@ -78,12 +90,11 @@ let myAks = aks {
         }
     ]
     linux_profile "aksuser" "public-key-here"
-    service_principal_client_id "some-spn-client-id"
+    service_principal_use_msi
```

Comment:
It doesn't like this this is a "new" keyboard but if it is, consider parameterisation of it with e.g. a boolean, which makes it easier for programmatically working with it.